avaframe/com1DFA/particleTools.py

```diff
@@ -0,0 +1,640 @@
+"""
+    Tools for handling particles, splitting, merging and tracking.
+"""
+
+# Load modules
+import logging
+import numpy as np
+import numbers
+import math
+
+# Local imports
+import avaframe.com1DFA.DFAtools as DFAtls
+
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def removePart(particles, mask, nRemove, reasonString=''):
+    """ remove given particles
+
+    Parameters
+    ----------
+    particles : dict
+        particles dictionary
+    mask : 1D numpy array
+        particles to keep
+    nRemove : int
+        number of particles removed
+    reasonString: str
+        reason why removing particles - for log message
+
+    Returns
+    -------
+    particles : dict
+        particles dictionary
+    """
+    if reasonString != '':
+        log.info('removed %s particles %s' % (nRemove, reasonString))
```

Comment:
so if reasonString is '' it does not print anything. But we want the log.info if the reason is for example: because they were outside of the release area or because they exited the domain

Reply:
And what is the consequence for an end / expert user (i.e. non-developer; the target audience for the info-level)?
If it is so severe that some action is required, we should elevate it to Warning or even Error...

Background: RAMMS gives this info as well, and everybody just simply ignores it.